lib/charts/charts.dart

```diff
@@ -69,4 +76,8 @@ abstract class LineChart<T> {
   bool get isMounted {
     return chartElement.element.parent != null;
   }
+
+  void dispose() {
```

Comment:
I don't think this is being called, but I presume that's because we currently never actually destroy this (only move it out/in the DOM when changing pages)?

Reply:
Yeah, it's not called, but having it makes it slightly more clear that the object needs cleanup.

I'll add an issue for onMounted() and onUnmounted() lifecycle events (as useful, but not an immediate concern).